### ISSUE TITLE
Migrate to Volto with plone.volto in upgrade guide needs details

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1559]
tl;dr: details about how to migrate to Volto from Classic UI are needed. The suggested one-liner in `[docs/backend/upgrading/version-specific-migration/migrate-to-volto.md](https://github.com/plone/documentation/pull/1423/files#diff-4ef60279d886f6ac59d7f725fb253d0a099a22940a9ceb8c990e47ee7ec9ff22) is a good start, but @fredvd offers further details that should be documented in that file.

---

I think I can give some more guidance here as well:

"Migrate to richtext" is only concerned with converting (a) richtext field(s) on selected or all content items in your site to the new blocks_layout field in the plone server where all contnt is stored.  That's why you can select content types to convert.

But to successfluly migrate your 'ClassicUI' site contents to the *default setup* that the Plone 6 ReactUI based (Volto) frontend requires, more needs to be done.   Such as: 

* Activate the plone.volto profile in your site
* Pages and Folders are merged to Folderish pages
* default views are converted by pulling the contents from the default page to the folderish page above
* Convert the Collection content type to a Folderish page with the collection/listing tile in the block layout
* AND convert the richtext fields to blocks layout

As you can see  "migrate to volto" does it all, migrate richtext is a part of if.  For custom setups, migrate to richtext allows you low level access to this tool . 

Why document the 'migate richtext' view?  Because if you are an integrator, developer or expert on migration, and don't want/need the default setup with the context type  it gives us flexibility in when to process the individual steps. 

@FarooqAlaulddin Is this an explanation that gives you more understanding, are there other questions that should be answered specifically to this topic?

@stevepiercy Do you want to update the paragraphs/section involved, or should I suggest an improvement after @FarooqAlaulddin gives his feedback?

_Originally posted by @fredvd in https://github.com/plone/documentation/issues/1423#issuecomment-1426739708_
